### DISCUSSION TITLE
v1.4

### DISCUSSION THREAD

**Discussion by buresdv** [buresdv/Cork#335] [Announcements]
## What's Changed
* Update Simplified Chinese localization for licensing branch by @Jerry23011 in https://github.com/buresdv/Cork/pull/286
* update zh_Hant by @sh95014 in https://github.com/buresdv/Cork/pull/287
* update zh_Hant for licensing by @sh95014 in https://github.com/buresdv/Cork/pull/289
* ~ German language by @hecaex in https://github.com/buresdv/Cork/pull/292
* Update german language by @hecaex in https://github.com/buresdv/Cork/pull/300
* Add a method to InstallationProgressTracker by @mattmassicotte in https://github.com/buresdv/Cork/pull/295
* Force dump to overwrite an existing Brewfile by @mjsrs in https://github.com/buresdv/Cork/pull/299
* Uploading Korean files  by @hjm79 in https://github.com/buresdv/Cork/pull/303
* korean translate 99% by @hjm79 in https://github.com/buresdv/Cork/pull/306
* Updated French translation by @Maxeuh in https://github.com/buresdv/Cork/pull/311
* + german strings by @hecaex in https://github.com/buresdv/Cork/pull/312
* Fix for French translation by @Maxeuh in https://github.com/buresdv/Cork/pull/316
* Delete .DS_Store by @mrvincenzo in https://github.com/buresdv/Cork/pull/317
* + Russian localization for new strings by @utkinn in https://github.com/buresdv/Cork/pull/324
* ~german strings by @hecaex in https://github.com/buresdv/Cork/pull/325
* + SC Localization by @Jerry23011 in https://github.com/buresdv/Cork/pull/326

## New Contributors
* @mattmassicotte made their first contribution in https://github.com/buresdv/Cork/pull/295
* @hjm79 made their first contribution in https://github.com/buresdv/Cork/pull/303
* @Maxeuh made their first contribution in https://github.com/buresdv/Cork/pull/311
* @mrvincenzo made their first contribution in https://github.com/buresdv/Cork/pull/317

**Full Changelog**: https://open.substack.com/pub/corkapp/p/cork-14-out-licensing-services-and?r=2vioyx&utm_campaign=post&utm_medium=web&showWelcomeOnShare=true

<hr /><em>This discussion was created from the release <a href='https://github.com/buresdv/Cork/releases/tag/v1.4'>v1.4</a>.</em>

**Comment by hutianlin2** [comment]
good
